src/lib/ledgers/multiledger.js

```diff
@@ -1,38 +1,53 @@
 'use strict'
 
 const FiveBellsLedger = require('./five-bells-ledger')
+const healthStatus = require('../../common/health.js')
 
 function Multiledger (options) {
   this.config = options.config
   this.log = options.log
   this.ledger_types = {undefined: FiveBellsLedger}
-  this.ledgers = {}
+  this.ledgers = this.buildLedgers()
+  this.ledgersHealth = { ledgersHealth: healthStatus.statusNotOk }
 }
 
 Multiledger.prototype.addLedger = function (Ledger) {
   this.ledger_types[Ledger.TYPE] = Ledger
 }
 
 Multiledger.prototype.getLedger = function (ledgerId) {
-  return this.ledgers[ledgerId] ||
-        (this.ledgers[ledgerId] = this.buildLedger(ledgerId))
+  return this.ledgers[ledgerId]
```

Comment:
In the old implementation, `this.ledgers` was populated lazily when `getLedger()` was called. In the new one, this is done in the ctor since `checkLedgersHealth()` needs to be aware of all ledgers.